test/Core.Test/Tokens/TestTokenable.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Bit.Core.Tokens;
+
+namespace Bit.Core.Test.Tokens
+{
+    public class TestTokenable : Tokenable
+    {
+        public bool ForceInvalid { get; set; } = false;
+        public override bool Valid
+        {
+            get => !ForceInvalid;
+        }
+    }
+
+    public class TestExpiringTokenable : ExpiringTokenable
+    {
+        private bool _forceInvalid;
+
+        public TestExpiringTokenable() : this(false) { }
+
+        public TestExpiringTokenable(bool forceInvalid)
+        {
+            _forceInvalid = forceInvalid;
+        }
+        protected override bool TokenIsValid() => !_forceInvalid;
```

Comment:
It doesn't seem like we are using the `forceInvalid`.

Reply:
We use this in the Expiring token tests to validate that expiring tokenables still honor the validitity of the underlying tokenable